docs/Makefile

```diff
@@ -13,7 +13,7 @@ help:
 
 glossary:
 	echo "\033[1mGenerating dictionary.rst...\033[0m"
-	python _attic/glossary_generator.py
```

Comment:
FYI @aofarrel 
Was this causing an error @bvizzier ?

Reply:
Yes.  It was erroring out on the python invocation.  Changing it to python3 allowed it to work.

Reply:
My hunch is that Ben's `python` points to python2, while mine points to python3. I changed mine to 3 a while back and forgot about it, so this was a good catch on Ben's part -- calling Python3 explicitly is the way to go.

As a side note, Mac OS is going to finally remove Python 2 in the near-ish future: https://forums.macrumors.com/threads/apple-finally-removing-python-2-in-macos-12-3.2332855/

Reply:
ok, good to know. 
I guess in the end, CI wins anyway

Reply:
macOS has already removed Python 2 and the /usr/bin/python symbolic link in the 12.3 update.   The official command is now /usr/bin/python3.

Reply:
Just checked .bashrc and I think I discovered why I didn't get the memo... oops!
<img width="582" alt="alias" src="https://user-images.githubusercontent.com/27784612/177874240-99e3c9ea-056e-444c-930d-5a8db8dc1563.png">

Reply:
Interesting, I use a Python 3 virtual environment, and it works with `python`. But going to defer to the more experienced Python developers.